listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx

```diff
@@ -450,36 +449,48 @@ export default class UserEntityChart extends React.Component<
                   <ul className="dropdown-menu" role="menu">
                     <li>
                       <a
-                        href=""
+                        href={`?${this.buildURLParams(1, "week", entity)}`}
```

Comment:
Some of these parameters are now duplicated, and in different places - the page number 1 is duplicated here and in the changeRange method. the range parameter ("week") is listed twice. Maybe this could be rewritten a bit so that we could have a single object (or search string?) that can be passed in to multiple places.
It does mean that changeRange, changeEntity and changePage are actually all the same method, just with some different parameters. Perhaps this could be simplified into a single method.